tutorials/AddingNewModel.md

```diff
@@ -0,0 +1,28 @@
+<h1>Adding New Model - Caffe</h1>
+
+1. For Setup Instructions. - https://github.com/Cloud-CV/Fabrik/blob/master/README.md
```

Comment:
Modify to: For setup instructions, look at the [README](https://github.com/Cloud-CV/Fabrik/blob/master/README.md)